From: Olivier Berger <obergix@debian.org>
Date: Sun, 9 Feb 2014 19:50:35 +0100
Subject: shebang

---
 v3/web_exec.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/v3/web_exec.py b/v3/web_exec.py
index ed44d36..0ea0e98 100644
--- a/v3/web_exec.py
+++ b/v3/web_exec.py
@@ -1,4 +1,4 @@
-# <INSERT YOUR VERSION OF PYTHON HERE AS A #! LINE>
+#!/usr/bin/python
 
 # Minimal CGI script for Online Python Tutor (v3), tested under Python 2 and 3
 
